SimpleAudioPlayer/Teddy.MvvmCross.Plugins.SimpleAudioPlayer/MvxSimpleAudioPlayerExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Teddy.MvvmCross.Plugins.SimpleAudioPlayer
+{
+    public static class SimpleAudioPlayerExtensions
+    {
+        public static void PlayOnRepeat(this IMvxSimpleAudioPlayer player)
+        {
+            player.Completion += new EventHandler(delegate (Object o, EventArgs a)
```

Comment:
Hm, makes sense about the reset function. Then I changed the API so that it makes only the registration/de-registration of handlers, without playing.

```
player.SetUpLooping();
...
player.Play();
player.Stop();
...
player.TearDownLooping()